docs/pipelines/functions.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 4
+title: "Functions"
+---
+
+# Functions
+Functions enable you to utilize filters (middleware) and pipe (model) functions directly within the WebUI. While largely compatible with Pipelines, these native functions can be executed easily within Open WebUI. Example use cases for filter functions include usage monitoring, real-time translation, moderation, and automemory. For pipe functions, the scope ranges from Cohere and Anthropic integration directly within Open WebUI, enabling "Valves" for per-user OpenAI API key usage, and much more. 
```

Comment:
This is great for now, but we do also have manifolds available to use as well. It's not nearly as common outside of the provider pipelines and functions.